source/transfer_list.rst

```diff
@@ -657,5 +660,120 @@ such that the last ACPI table in this entry ends at offset
      - hdr_size
      - One or more ACPI tables.
 
+.. _optee_pagable_part_entry:
```

Comment:
I think it would be good to start a new sub-section for the OP-TEE TEs.

Reply:
Done.